CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+BUG FIXES:
```

Comment:
@sumovishal I didn't know where to put it. It can also be classified as enhancement (?).

Reply:
Let's put them under "ENHANCEMENTS".